.github/workflows/release.yml

```diff
@@ -291,10 +291,20 @@ jobs:
         uses: actions/download-artifact@v1
         with:
           name: jar
-      - run: |
+      - name: Run tests
+        if: runner.os != 'Windows'
+        run: |
           javac -cp "../jar/oso-${{ steps.version.outputs.oso_version }}.jar:." Test.java
           java -cp "../jar/oso-${{ steps.version.outputs.oso_version }}.jar:." Test
         working-directory: test
+      - name: Run tests
+        if: runner.os == 'Windows'
+        env:
+          github_ref: ${{ github.ref }}
+        run: |
+          javac -cp "..\jar\oso-$($env:github_ref.Substring(11)).jar;." Test.java
```

Comment:
Wait what is this doing?

Reply:
Pulling the version number off the `github_ref` environment variable which is something like `refs/tags/v0.0.0`

Reply:
Ahhh, okay from index 11 onwards. Makes sense.